### PR TITLE
Update Support_services_agreement.md

### PR DESCRIPTION
**Pull request by michelhofmans**
Better to name this page 'terms' instead of 'agreement'.

### CODE REVIEW

File: dataminer-overview/Support Services/Support_services_agreement.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ uid: Support_services_terms
...
 
-This document sets out the terms of Skyline's Support Services for its Software (the "**Agreement**").  
+This document sets out the terms of Skyline's Support Services for its Software (the "**Terms**").
 
 These terms apply on all orders for Support Services placed at Skyline by the User. User accepts these terms when placing Support Services orders at Skyline. Any deviations from these terms shall only be valid upon written agreement by Skyline, and any User terms on the same subject matter shall be deemed null and void.
 
 ## Definitions
 
-For the purpose of this Agreement, the following terms shall have the meanings indicated below:  
+For the purpose of this Terms, the following terms shall have the meanings indicated below:
```
Comment (michelhofmans) [inline dataminer-overview/Support Services/Support_services_agreement.md:15]: *these terms?

**Review by michelhofmans — COMMENTED** [review]
Looks good Marieke, thank you!

File: dataminer-overview/Support Services/Support_services_agreement.md (hunk 1 above)
Comment (MariekeGO) [reply dataminer-overview/Support Services/Support_services_agreement.md]: Corrected!

**Review by michelhofmans — COMMENTED** [review]
perfect, thanks

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**